lib/manageiq/appliance_console/prompts.rb

```diff
@@ -137,6 +138,10 @@ def ask_for_password(prompt, default = nil)
       pass == "********" ? (default || "") : pass
     end
 
+    def ask_for_region(prompt, default = "us-east-1")
```

Comment:
I don't think this should be in a separate `Prompts` method. I would prefer you use `just_ask("Amazon Region for database file", "us-east-1")` in the `DatabaseAdmin` class